### PR TITLE
created jobs controller and added 7 actions

### PR DESCRIPTION
**Pull request by contatc2**
created jobs controller with 7 actions - we should create the routes together

### CODE REVIEW

File: app/controllers/jobs_controller.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
+class JobsController < ApplicationController
+  before_action :set_requester, only: %i[new create]
+  before_action :find_job, only: %i[show edit update destroy]
+  def index
+    @jobs = Job.all
+  end
+
+  def show
```
Comment (OmarBerrada) [inline app/controllers/jobs_controller.rb]: why is this method empty?

File: app/controllers/jobs_controller.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    @job = Job.new
+  end
+
+  def create
+    @job = Job.new(job_params)
+    if @job.save
+      redirect_to requester_path(@requester)
+  end
+
+  def edit
```
Comment (OmarBerrada) [inline app/controllers/jobs_controller.rb]: why is the method empty?

File: app/controllers/jobs_controller.rb (hunk 1 above)
Comment (contatc2) [reply app/controllers/jobs_controller.rb]: the definition is set in the before action

File: app/controllers/jobs_controller.rb (hunk 2 above)
Comment (contatc2) [reply app/controllers/jobs_controller.rb]: same reason

**Pull request merged**